### PR TITLE
adding deploy preprod script

### PR DESCRIPTION
**Pull request by 23will**
seems to be missing

### CODE REVIEW

File: deploy/deploy_preprod.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+##run gulp before running deploy
```
Comment (frosforever) [inline deploy/deploy_preprod.sh]: I know this is the same as the other scripts but does it make sense to enforce this in a stronger way than just a comment in the script that gets run (not read). Should it print out on screen, ask a question, directly run gulp etc?

File: deploy/conf/env-preprod.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+//env-dev.js
```
Comment (frosforever) [inline deploy/conf/env-preprod.js]: `//env=preprod.js`

File: deploy/deploy_preprod.sh (hunk 1 above)
Comment (23will) [inline deploy/deploy_preprod.sh]: it does but not in this PR

**Comment by JimmyKuruvilla** [comment]
Our ui deploy script for insights copies preprod over to prod. Should this do the same? 

We also zip the web directory and add it as part of a release. Should we do that here? Does it make sense to do at all in either ui given that we could just insert the release commit sha instead?

**Comment by 23will** [comment]
@JimmyKuruvilla 

1. yes
2. yes, yes (as discussed in video call)

File: deploy/deploy_preprod.sh (hunk 1 above)
Comment (frosforever) [inline deploy/deploy_preprod.sh]: @23will is there another PR or issue?

**Comment by frosforever** [comment]
👍 LGTM

**Pull request merged**